igcollect/linux_disklatency.py

```diff
@@ -107,5 +87,45 @@ def main():
             print(template.format(mode, k, int(v)))
 
 
+def get_path(args) -> str:
+    if args.path is not None:
+        return args.path
+
+    # no --path is specified, use libvirt to get the storage volume
+    if args.storagepool is None or args.storagevol is None:
+        print('you have to set --storagepool AND --storagevol')
```

Comment:
You could (try to) import the libvirt module only here if required. This would raise an exception if the module is not present which you want in this case because there would be no point of continueing in this case.